tools/get_config.py

```diff
@@ -48,12 +48,12 @@
 
     # Target
     if options.mcu is None :
-        args_error(parser, "[ERROR] You should specify an MCU")
+        args_error(parser, "argument -m/--mcu is required")
```

Comment:
> args_error already prints an error message.


Yes, that's obvious from the outputs you shared but it changes from ``[ERROR]`` to ``command: error:``. I am not aware though how was that used (if there any tool that was parsing this or was just an unification of messages we print to have a format. That was my concern, looks fine to me as it is now ;)